mopidy_beetslocal/library.py

```diff
@@ -136,8 +136,9 @@ def browse(self, uri):
         elif level == "album":
             for track in self._browse_track(query):
                 result.append(Ref.track(
-                    uri="beetslocal:track:%s:%s" % (track.id,
-                                                    track.path.decode('utf8')),
+                    uri="beetslocal:track:%s:%s" % (
+                        track.id,
+                        uriencode(track.path.decode('utf8'), '/')),
```

Comment:
And, the `.decode('utf-8')` will also fail with a `UnicodeDecodeError` if there's non-UTF-8 data in the path. The encoding of the file system usually follows the computer's/user's locale, and it will often be something else than UTF-8.

In other word, throw away the old decode, and use your new uriencode alone. It's way better because it keeps the exact bytes from the original path bytestring.